src/templates/BlogPost/index.css

```diff
@@ -226,49 +227,51 @@
     height: 400px;
     margin: $spacing-84 0;
   }
-}
 
-:global {
-  .gatsby-resp-image-figure {
-    margin: $spacing-52 0;
+  :global {
+    .gatsby-resp-image-figure {
```

Comment:
Were those spaces intentional?

Reply:
Idk, I just moved these rules into the `.content` scope and ran the `stylelint` after that :man_shrugging:

Reply:
Perhaps it's github making a big mess. I just moved this inside the bigger `.content` rule so we only apply these rules there.